docs/tutorials/PipelinesApi101.md

```diff
@@ -74,9 +75,9 @@ Create a <a href="https://cloud.google.com/resource-manager/docs/creating-managi
 
 On your Google project, open up the <a href="https://console.developers.google.com/apis/library" target="_blank">API Manager</a> and enable the following APIs:
         
-* Google Compute Engine
-* Google Cloud Storage
-* Genomics API
+* Google Compute Engine API
```

Comment:
From searching the API library, these seem to be the correct names. However, my project already had all the APIs enabled, so I can't be sure that these are the only ones needed.